app/assets/javascripts/components/instructor_panel.js.jsx

```diff
@@ -56,8 +66,15 @@ var InstructorPanel = React.createClass({
         <Action data={this.getInstructorToggleButtonData()} />
         <Action data={this.getTakeQueueOfflineButtonData()} />
         <Action data={this.emptyQueueButtonData()} />
-        
-
+        <div className="ui bottom padded fluid large menu">
+          <div className="ui simple dropdown item">
```

Comment:
I think making this a `ui fluid simple dropdown item` would accomplish the same thing as the CSS you added for the `dropdown` class?

Reply:
Yup, removed css for `dropdown` in favor of `ui fluid simple dropdown item`.